### PR TITLE
Cally's Trials

### PR DESCRIPTION
**Pull request by MadShmupper**
# New Port for { Cally's Trials }

## Game Information
- **Title**: { Cally's Trials }
- **URL**: { https://store.steampowered.com/app/476510/Callys_Trials/ }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [ ] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by Slayer366 — APPROVED** [review]

File: ports/callystrials/callystrials/libs/libcrypto.so.1.1
Comment (cdeletre) [inline ports/callystrials/callystrials/libs/libcrypto.so.1.1:1]: please add the [license](https://github.com/PortsMaster/PortMaster-New/blob/main/ports/minidoom2/minidoom2/licences/LICENSE.libcrypto.txt)

File: ports/callystrials/callystrials/libs/libzip.so.5
Comment (cdeletre) [inline ports/callystrials/callystrials/libs/libzip.so.5:1]: Please add the [license](https://github.com/PortsMaster/PortMaster-New/blob/main/ports/minidoom2/minidoom2/licences/LICENSE.libzip.txt)

File: ports/callystrials/callystrials/gamedata/splash.png
Comment (cdeletre) [inline ports/callystrials/callystrials/gamedata/splash.png:1]: Is it the splash we want ? a white square ? :)

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/callystrials/callystrials/libs/libcrypto.so.1.1
Comment (MadShmupper) [reply ports/callystrials/callystrials/libs/libcrypto.so.1.1:1]: Added. Thank you🎩

File: ports/callystrials/callystrials/libs/libzip.so.5
Comment (MadShmupper) [reply ports/callystrials/callystrials/libs/libzip.so.5:1]: Good to go. I also added them to my pr of Tamashii.🙂

File: ports/callystrials/callystrials/gamedata/splash.png
Comment (MadShmupper) [reply ports/callystrials/callystrials/gamedata/splash.png:1]: I took it out. It wasn't really needed. I had it in there as a gitkeeper type file and never took it out. Thank you🎩

**Review by Cebion — DISMISSED** [review]

**Review by cdeletre — DISMISSED** [review]

**Pull request merged**